react_juce/core/ImageView.cpp

```diff
@@ -120,7 +164,9 @@ namespace reactjuce
                 if (!juce::URL(source).readEntireBinaryStream(mb)) 
                 {
                     shouldDownloadImage = true;
```

Comment:
Hm, realizing this, shouldn't we not need this here? This would essentially mean "keep trying to download it" if I understand correctly? We could in theory end up in an endless loop trying to reach the same server and constantly failing, no?